Update README file (#102)
Luca1991/xbox_kernel_test_suite: README.md

# xbox kernel test suite
[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/blob/master/LICENSE)
[![GitHub CI](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/actions/workflows/ci.yml?query=branch%3Amaster)

Xbox kernel APIs tester written using the open-source nxdk.

> This is a tool for testing the xbox kernel APIs on the hardware to verify tests are passing. Plus we have a dedicated wiki page for [homebrew kernels](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/wiki/List-of-Homebrew-Kernels-Tested) have used this tool to test their kernel implementation.

Reviewer: ```suggestion
This is a tool for testing the xbox kernel APIs on the hardware to verify their workings. In addition, we have a dedicated wiki page for [homebrew kernels](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/wiki/List-of-Homebrew-Kernels-Tested) that have used this tool to test their kernel implementation.
```

Author: fixed ~~, but replaced "their workings" to "they're working".~~

## HOW TO BUILD:
All you need is nxdk. You can get it here: https://github.com/XboxDev/nxdk

Here is a setup guide: https://github.com/XboxDev/nxdk/wiki/Getting-Started

## CONFIGURATION FILE:
> [!NOTE]
> The configuration file should be called _**"config.txt"**_ and should be placed in the same directory of the xbe.

> The following list of options can be used inside config.txt file:

Reviewer: ```suggestion
The following list of options can be used inside the config.txt file:
```

Author: fixed

> - `seed` = `<hexadecimal (support up to unsigned int max)>`

Reviewer: I'd suggest to be explicit here and state that the maximum allowed value for the seed is 0xFFFFFFFF, just like it's done below.

Author: fixed

- `tests` = `<hexadecimal (support up to 17A)>[,...]`
- `tests-exclude` = `<hexadecimal (support up to 17A)>[,...]`
- `disable-video` = `<boolean>`[^1]

> [^1]: boolean value can be 1 or 0

Reviewer: This is not true. The code for the disable-video option is using a `if` statement to check if it should output to the video. This means that any value other than zero will not output, not just one like this line claims.

Author: Technically, 1-9A-F characters will be read as "true" which then will make video disable due to inverter. If no input or zero will be treat as video enable by default.

But this is more or less a guideline for readers should follow properly.
